### ISSUE TITLE
Collection : check image resolution

### ISSUE THREAD

**Issue by pacomski** [communicationartetmarges/art-et-marges#1982]
@sarahkokot : vérifier si la résolution des images est suffisante ou non.

**Comment by FredericOszczak** [comment]
oui, entre-temps le problème avait été réglé. Il y a juste certains artistes qui n'ont pas d'image dans leur fiche. Mais je reviendrai avec cela, quand j'ajouterai de nouveaux contenus ici.
